cluster/proto/cluster/message.proto

```diff
@@ -2,48 +2,40 @@ syntax = "proto3";
 
 package weaviate.cloud.internal.cluster;
 
-//option go_package = "github.com/weaviate/weaviate/cloud/proto/cluster";
+service ClusterService {
```

Comment:
Re-orged the file to have the Service on top and Request/Response pair sequential in the file.
Just added the new endpoint `Query`